### ISSUE TITLE
BUG: Unhandled exception during task queue run loop.

### ISSUE THREAD

**Issue by effemmeffe** [Flexget/Flexget#1026]
2016-04-11 21:02 CRITICAL task_queue                    BUG: Unhandled exception during task queue run loop.
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flexget/task_queue.py", line 47, in run
    self.current_task.execute()
  File "/usr/local/lib/python2.7/dist-packages/flexget/task.py", line 69, in wrapper
    return func(self, *args, **kw)
  File "/usr/local/lib/python2.7/dist-packages/flexget/task.py", line 579, in execute
    self._execute()
  File "/usr/local/lib/python2.7/dist-packages/flexget/task.py", line 545, in _execute
    self.__run_task_phase(phase)
  File "/usr/local/lib/python2.7/dist-packages/flexget/task.py", line 409, in __run_task_phase
    if phase == 'input' and response:
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/list/trakt_list.py", line 91, in __len__
    return len(self.items)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/list/trakt_list.py", line 140, in items
    raise plugin.PluginError('Could not retrieve list from trakt (%s)' % e.args[0])
PluginError: Could not retrieve list from trakt (HTTPSConnectionPool(host='api-v2launch.trakt.tv', port=443): Read timed out. (read timeout=30))

**Comment by liiight** [comment]
You got a timeout from trakt. This shouldn't crash the task, did it?

**Comment by effemmeffe** [comment]
That was exactly my question.

**Comment by liiight** [comment]
hmm, let me rephrase.
**Did** it crash flexget?

**Comment by effemmeffe** [comment]
How do I know?
I found that line in the log and that traceback in a crash file.

**Comment by liiight** [comment]
if the log didn't stop there, flexget didn't crash.
maybe a crash report for something like a timeout is too severe, perhaps it'll be better just to log an error

**Comment by effemmeffe** [comment]
The log didn't stop.

**Comment by gazpachoking** [comment]
This is definitely a bug and should be handled. We shouldn't be having tracebacks in normal operation.
